dist/ember-json-api.js

```diff
@@ -319,9 +373,9 @@ define("json-api-adapter",
        * Use "links" key
        */
       serializeHasMany: function(record, json, relationship) {
-        var attr = relationship.key;
-        var type = this.keyForRelationship(relationship.type.typeKey);
-        var key = this.keyForRelationship(attr);
+        var attr = relationship.key,
+          type = this.keyForRelationship(relationship.type.typeKey),
+          key = this.keyForRelationship(attr);
```

Comment:
Do you mind not changing the style in this PR? Nothing against the change, but I'd rather do it in a separate PR so that we can discuss it specifically.

Using `var` in each line makes it easier to debug stuff.